frontend/.lintstagedrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "*.{ts,tsx,js,jsx,scss,css,md}": "prettier --write",
+  "*.{ts,tsx,js,jsx}": (filenames) =>
+    `next lint --fix --file ${filenames
+      .map((file) => file.split(process.cwd())[1])
+      .join(" --file ")}`,
+};
```

Comment:
This is also good to call out: when you run `npm install` in `/frontend`, this adds a Git hook that runs Prettier and ESLint on any changes you're about to commit. Not sure if that (especially Prettier) is something we want.

Reply:
From other comment, if we can migrate to Stylelint, lets. I want to use the same tools everywhere we can.

Reply:
Stylelint and Prettier do different (though sometimes conflicting) things, so we can use both if we want. My comment here was mostly husky and how it modifies your local clone's Git hooks though, since that can be unexpected, and apparently has caused issues for Greg before. But we can remove it when that happens :)